terraform/modules/platform/outputs.tf

```
@@ -148,3 +148,8 @@ output "network_access_logs_bucket" {
   description = "Standardized CMS Hybrid Cloud Providued Network Access Logs bucket Name"
   value       = "cms-cloud-${data.aws_caller_identity.this.account_id}-${data.aws_region.primary.name}"
 }
+
+output "cloudfront_logs_bucket_arn" {
```

Since this bucket also contains logs for load balancers that get ingested in Splunk, I think sticking with something closer to the name you used for the data call would make sense


```suggestion
output "splunk_logging_bucket_arn" {
```

---

Ok, renamed and updated. Started a branch in ab2d that has the matching update. Will submit PR for that once the REF # is out for this update.